### PR TITLE
Rest of roles function done + translation

### PR DESCRIPTION
**Pull request by Andersengg**

### CODE REVIEW

File: HGGM/Views/Roles/Details.cshtml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    @Model.NormalizedName
+                </td>
+                <td>
+                    @Model.Permissions
+                </td>
+
+            </tr>
+        </tbody>
+    </table>
+    <dl class="dl-horizontal"></dl>
```
Comment (j2ghz) [inline HGGM/Views/Roles/Details.cshtml]: What is this for?

File: HGGM/Views/Roles/Edit.cshtml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+<h4>@Model.Name</h4>
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        <form asp-action="Edit">
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <div class="form-group">
+                <label asp-for="Id" class="control-label"></label>
+                <input asp-for="Id" class="form-control" />
```
Comment (j2ghz) [inline HGGM/Views/Roles/Edit.cshtml]: `form-control` is bootstrap

File: HGGM/Views/Roles/Details.cshtml (hunk 1 above)
Comment (Andersengg) [reply HGGM/Views/Roles/Details.cshtml]: removed (fixed)

File: HGGM/Views/Roles/Edit.cshtml (hunk 2 above)
Comment (Andersengg) [reply HGGM/Views/Roles/Edit.cshtml]: Changed (fixed)

File: HGGM/Views/Roles/Details.cshtml [hunk 3]
```diff
@@ -48,13 +47,11 @@
...
                     @Model.Permissions
                 </td>
-
             </tr>
         </tbody>
     </table>
-    <dl class="dl-horizontal"></dl>
 </div>
 <div>
     @Html.ActionLink("Edit", "Edit", new { id = Model.Id }) |
```
Comment (j2ghz) [inline HGGM/Views/Roles/Details.cshtml]: `Edit` needs to be translatable

**Pull request merged**